mvc.md

```diff
@@ -0,0 +1,123 @@
+# Psuedo Code
+
+
+## Model
+- Update the model fields to match the front end requirements.
+- Does education have any functions?  No, only for ReportModel. those functions talk directly to database AND/OR helpers
+
+doesn't have a user field...once they sign in
+
+## FrontEnd Requirements
```

Comment:
Great, this lines up with what we have as our data model. Thanks!

Reply:
@timkeresey: Please be advised: we made some minor updates that will be exposed in the next couple of PR's as we realized how the data would come through after json and after interacting with our model.